### ISSUE TITLE
Maquette desktop - Enjoy it :)

### ISSUE THREAD

**Issue by SirBeerus1** [anisayari/createurstech.fr#37]
Yo,

Chose promise, chose dûe : 
Je ne savais pas où la mettre mais, voici la **première version de la maquette** desktop.

[-> Artboard pour récup les assets ](https://www.figma.com/file/i7PgkummQdJhcPEsGUhsa0/Cr%C3%A9ateursTech.fr?node-id=28%3A2911)

[-> Simulation clickable (surbrillances en bleu / "R" pour restart)](https://www.figma.com/proto/i7PgkummQdJhcPEsGUhsa0/Cr%C3%A9ateursTech.fr?page-id=0%3A1&node-id=1%3A2&viewport=-315%2C331%2C0.4865090250968933&scaling=min-zoom)

Résultats de 2 nuits blanches, mais j'ai pris du kiff à le faire ! Hâte de voir ce que ça va donner ! 
N'étant pas un pur UIdesigner, je suis à l'écoute de toutes vos remarques ;) 
_(DM sur Discord / Twitch)_

**A dispo. pour la V2 !**
++

**Comment by anisayari** [comment]
Oh wow genial !!!!

**Comment by Gr0bs** [comment]
Vraiment cool ! 
Je mettrai peut-être une seconde lignes avec les résultats de recherche pour éviter que si il y a 30 card que ça soit pas trop long à parcourir.

**Comment by DimitriTimoz** [comment]
@anisayari ce serait bien je pense d'attribuer la tache de l'implémentation à quelqu'un histoire de ne pas être plusieurs dessus. 
Personnellement je veux bien m'en occuper.

**Comment by anisayari** [comment]
> @anisayari ce serait bien je pense d'attribuer la tache de l'implémentation à quelqu'un histoire de ne pas être plusieurs dessus. 
> Personnellement je veux bien m'en occuper. 

Ok go. Tu peux te charger des cartes ? Il n'y a que les image de profil par contre. Pas d'image de couverture. 
Il n'y a pas la date de la dernière vidéo aussi. 

Et pour l'instant on va garder la forme de grid. Plutôt que la recherche de scroll horizontal. On va y aller petit à petit.

**Comment by DimitriTimoz** [comment]
Par contre isotope va être énervant pour le carrousel car il gère le positionnement avec une position absolue. Il faudra soit trouver un moyen de désactiver ce positionnement par isotopes ou refaire le système de filtres.

**Comment by anisayari** [comment]
> Par contre isotope va être énervant pour le carrousel car il gère le positionnement avec une position absolue. Il faudra soit trouver un moyen de désactiver ce positionnement par isotopes ou refaire le système de filtres. 

Oui c'est pour ça qu'on va y aller petit à petit et garder isotope pour l'instant.

**Comment by DimitriTimoz** [comment]
> > Par contre isotope va être énervant pour le carrousel car il gère le positionnement avec une position absolue. Il faudra soit trouver un moyen de désactiver ce positionnement par isotopes ou refaire le système de filtres.
> 
> Oui c'est pour ça qu'on va y aller petit à petit et garder isotope pour l'instant.

aucun problème, je fais ça

**Comment by DimitriTimoz** [comment]
@anisayari sur la carte il y a un système de votes, je l'implémente ?

**Comment by anisayari** [comment]
> @anisayari sur la carte il y a un système de votes, je l'implémente ?

Non. C'est une bonne idée. Mais Ça va être spammer de bot click ça c'est sur sans authentification.

**Comment by DimitriTimoz** [comment]
l'arrière plan de la carte ne rend pas bien sur le fond noir, je peux changer aussi le background ?

**Comment by anisayari** [comment]
> @anisayari  l'arrière plan de la carte ne rend pas bien sur le fond noir, je peux changer aussi le background ?

Oui go

**Comment by Barbapapazes** [comment]
Yo les gars ! Super maquette mais ça serai hyper intéressant d'avoir aussi la version mobile pour commencer par intégrer celle là et pas se trouver bloqué par la suite ! Ensuite, j'ai vu des boutons pour up vote ou down vote mais il y a un serveur qui gère ça ?

En tout cas, c'est propre !

**Comment by SirBeerus1** [comment]
Yo l'équipe ! 
Merci pour vos retours, ça fait plaisir :) 

Pas mal de commentaires intéressants, je voudrais attaquer la version mobile, mais avant tout faut qu'on soit sûr de la version desktop avant de la duppliquer vers un autre format. 
Mais, si on bouge pas énormément, il ne devrait pas ya avoir beaucoup de changements. 

Je propose de tenir un [Kanban sur Notion](https://www.notion.so/edbc5847833f43079b1c3d488e7350c1?v=59bd76ef33434e508912489845d9c019) pour ajouter/suivre/gérer les modifs.. 

Pour revenir sur ces points : 

**- Carousel Cards :** J'imaginais un carousel d'environ 10 cards par ligne. Si dépasse, une nouvelle rangée de cards viendrait s'ajouter au-dessous, ainsi de suite. 
Mais en effet, une chose à la fois, au besoin, je peux revoir la disposition en version Grid ?

**- Cards :** -> Dommage pour la perte de la bannière (youtube ou peu importe), c'était pour appuyer le branding de chaque univers des créateurs. 
-> La feature : date du dernier feed (youtube/twitter/twitch) du créateur était pour donner de l'envie d'en savoir plus sur la personne. Notamment un critère de recherches, mais j'imagine que ça doit être chaud à coder. 
-> Les champs filtres seront quand même intégrés comme dans la search bar, ou je redesign dans le même style que sur le live ?

**- Feature "Votes" :** Petite initiative de ma part pour ajouter un nouveau use case pour l'utilisateur qui donne force de référence au site et donne envie d'y revenir. En effet plus safe si on a son propre profil. Feature future ?
